test-files/io/ServiceHootApiDbReaderTest/runReadByBoundsTestOutput.osm

```diff
@@ -2,82 +2,67 @@
 <osm version="0.6" generator="hootenanny" srs="+epsg:4326">
     <node visible="true" id="2" version="1" lat="33.9008974999999992" lon="-83.0226542999999992">
         <tag k="node6" v="node6"/>
-        <tag k="error:circular" v="15"/>
```

Comment:
In this particular instance, the original data doesn't have `error:circular` in it so the value of `15` is the default value that was being forced in there by the software (i.e. set to `15` instead of `-1`) when it wasn't necessarily needed.  See `ElementData::hasCircularError()`